src/browser/src/app/components/extension-select/export-api/export-api.component.ts

```diff
@@ -32,16 +33,15 @@ export class ExportApiComponent implements OnInit {
   ) {}
   ngOnInit(): void {
     this.initData();
-    this.messageService
-      .get()
-      .pipe(takeUntil(this.destroy$))
-      .subscribe((inArg: Message) => {
-        if (inArg.type === 'extensionsChange') {
-          this.initData();
-        }
-      });
+    this.watchInstalledExtensionsChange();
+  }
+
+  @ExtensionChange('exportAPI')
```

Comment:
you can delete this function `watchInstalledExtensionsChange`.
Add ExtensionChange at `initData` function